### ISSUE TITLE
Don't clone some elements at startup

### ISSUE THREAD

**Issue by nhuillard** [sutara79/jquery.add-input-area#8]
I needed a way to preview already uploaded files (with `<input type=file>`), but obviously omit the preview tags for the new elements created when adding a row of new inputs. In order to do that, I added a `dont-clone` class to those elements, triggering `detach()` (see lines 59-60 below). I didn't add configuration settings for that though...

```
 57   // Save original
 58   this.option.original = $(this.elem).find(this.option.area_var).eq(0).clone(this.option.clone_event);
 59   // Remove dont-clone elements
 60   $(this.option.original).find('.dont-clone').detach();

```

**Comment by sutara79** [comment]
I am sorry for the delay in my response.
I will add new option `dont_clone` at next update (v4.10.0).
Thank you.
